pkgs/applications/graphics/geeqie/default.nix

```diff
@@ -1,23 +1,24 @@
-{ stdenv, fetchurl, pkgconfig, autoconf, automake, gtk2, libpng, exiv2
-, lcms, intltool, gettext, fbida
+{ stdenv, fetchurl, pkgconfig, autoconf, automake, gettext, intltool
+, gtk3, lcms2, exiv2, libchamplain, clutter_gtk, ffmpegthumbnailer, fbida
 }:
 
 stdenv.mkDerivation rec {
   name = "geeqie-${version}";
-  version = "1.3";
+  version = "1.4";
 
   src = fetchurl {
     url = "http://geeqie.org/${name}.tar.xz";
-    sha256 = "0gzc82sy66pbsmq7lnmq4y37zqad1zfwfls3ik3dmfm8s5nmcvsb";
+    sha256 = "0ciygvcxb78pqg59r6p061mkbpvkgv2rv3r79j3kgv3kalb3ln2w";
   };
 
-  preConfigure = "./autogen.sh";
+  # Do not build the changelog as this requires markdown.
+  patches = [ ./geeqie-no-changelog.patch ];
 
-  configureFlags = [ "--enable-gps" ];
+  preConfigure = "./autogen.sh";
```

Comment:
I think you are supposed to add `NOCONFIGURE=1` before `./autogen.sh`. That way configuration will actually be done during `configurePhase`.

Reply:
Looks like it. The `--enable-gps` flag however was renamed and enabled by default anyway which is why I removed it. It would be probably best to still add `NOCONFIGURE` in case someone would want to enable or disable a feature in the future.

Reply:
`NOCONFIGURE` just prevents, running `./configure` from `./autogen.sh`, which is not needed since it will be called in `configurePhase`.